src/video/windows/SDL_windowsevents.c

```diff
@@ -2601,14 +2600,17 @@ void WIN_PumpEvents(SDL_VideoDevice *_this)
         }
     }
 
-    // Update the clipping rect in case someone else has stolen it
+    // fire queued clipcursor refreshes
     if (_this) {
         SDL_Window *window = _this->windows;
         while (window) {
             SDL_WindowData *data = window->internal;
-            if (data && data->skip_update_clipcursor) {
-                data->skip_update_clipcursor = false;
-                WIN_UpdateClipCursor(window);
+            if (data) {
+                data->postpone_clipcursor = false;
```

Comment:
Can move this assignment into the if statement, it's only set to true iff clipcursor_queued is true